### PR TITLE
Add new yosys runner

### PR DESCRIPTION
**Pull request by wtatarski**
PR adds yosys as a runner from this repo: https://github.com/antmicro/yosys/tree/ibex-wip-symbiflow

### CODE REVIEW

File: tools/runners.mk [hunk 1]
```diff
@@ -21,6 +21,13 @@ $(INSTALL_DIR)/bin/yosys:
 	$(MAKE) -C $(RDIR)/yosys ENABLE_TCL=0 ENABLE_ABC=0 ENABLE_GLOB=0 ENABLE_PLUGINS=0 ENABLE_READLINE=0 ENABLE_COVER=0
 	install -D $(RDIR)/yosys/yosys $@
 
+# yosys-sv
+yosys-sv: $(INSTALL_DIR)/bin/yosys-sv
+
+$(INSTALL_DIR)/bin/yosys-sv:
+	$(MAKE) -C $(RDIR)/yosys-sv ENABLE_TCL=0 ENABLE_ABC=0 ENABLE_GLOB=0 ENABLE_PLUGINS=0 ENABLE_READLINE=0 ENABLE_COVER=0
```
Comment (tgorochowik) [inline tools/runners.mk:28]: Could we only add the `third_party/cores/ibex-yosys-build` submodule and build yosys from the submodule there? I think there is no point in keeping it as our submodule directly

File: tools/runners/Yosys_sv.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            for svf in params['files']:
+                f.write('read_verilog -sv' + inc + ' ' + svf + '\n')
+
+    def get_version_cmd(self):
+        return [self.executable, "-V"]
+
+    def get_version(self):
+        version = super().get_version()
+
+        return " ".join([self.name, version.split()[1]])
```
Comment (tgorochowik) [inline tools/runners/Yosys_sv.py:32]: Could we append a short note to the version explaining what this runner exactly is?

**Comment by kgugala** [comment]
Conda PR is now merged

**Comment by kgugala** [comment]
conda package has been built and deployed, restarted CI here

**Pull request merged**